Readmes/Hemant/README.md

```diff
@@ -0,0 +1,60 @@
+</p>
+
+<br/>
+
+## 📖  What I am currently learning / improving on
+<p align="center">
+<img src="https://img.shields.io/badge/Vuejs-282C34?logo=Vue.js&logoColor=43B882" alt="Vuejs logo" title="Vue.js" height="25" />
+<img src="https://img.shields.io/badge/Redux-282C34?logo=redux&logoColor=764ABC" alt="Redux logo" title="Redux" height="25" />
+ </p>
+ 
+<br/>
+ 
+## 🔥 Streak Stats
+<p align="center"><img src="https://github-readme-streak-stats.herokuapp.com/?user=hemantwasthere&theme=black-ice" alt="Salonee-Jain"  /></p>
+
+<br>
+
+## 📬 Get in touch
+
+<p align="center">
+<a href="https://twitter.com/hemantwasthere" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/twitter.svg" alt="twitter" height="30" width="40" /></a>
+<a href="https://linkedin.com/in/hemantwasthere" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="linkedin" height="30" width="40" /></a>
+<a href="https://instagram.com/hemantwasthere" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/instagram.svg" alt="instagram" height="30" width="40" /></a>
+</p>
+
+<br>
+
+## 😂 Here is a random joke that'll make you laugh!
+![Jokes Card](https://readme-jokes.vercel.app/api)</center>
+
+<details>
+<summary><strong>More info..</strong></summary>
+
+## &#x1f4c8; My GitHub Stats
+<p  align="center"> 
+<a href="https://github.com/hemantwasthere">
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=hemantwasthere&show_icons=true&theme=algolia&include_all_commits=true&count_private=true"/>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=hemantwasthere&theme=algolia&hide=c%2B%2B&layout=compact"/>
+</a>
+</p>
+<img width="100%" src="https://activity-graph.herokuapp.com/graph?username=hemantwasthere&bg_color=000000&color=1fdbd8&line=ff5c5c&point=1adbce&area=true&hide_border=false" />
+
+## 🧋 Support
```

Comment:
Hmm, I cannot see this emoji. Would you like to show a screenshot of how it looks?
```suggestion
## 🧋 Support

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient